docs/src/appendix/scarb-toml.md

```diff
@@ -78,6 +78,15 @@ url = "http://your.second.rpc.url"
 block_id.number = "123"
 ```
 
+### `[tool.scarb]`
+
+```toml
+[tool.scarb]
+allow-prebuilt-plugins = ["snforge_std"]
+```
+It allow `scarb` to download precompiled dependencies used by `snforge_std`.
+The `snforge_std` library depends on a Cairo plugin that is written in Rust and, by default, is compiled locally on the user's side.
```

Comment:
```suggestion
The `snforge_std` library depends on a Cairo plugin that is written in Rust, and otherwise is compiled locally on the user's side.
```

maybe?